test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -23,8 +23,8 @@ hw.module @M1<param1: i42>(%clock : i1, %cond : i1, %val : i8) {
   // CHECK-NEXT:   `ifndef SYNTHESIS
     sv.ifdef.procedural "SYNTHESIS" {
     } else {
-  // CHECK-NEXT:     if (PRINTF_COND_ & 1'bx & 1'bz & 1'bz & cond & forceWire)
-      %tmp = sv.verbatim.expr "PRINTF_COND_" : () -> i1
+  // CHECK-NEXT:     if (`PRINTF_COND_ & 1'bx & 1'bz & 1'bz & cond & forceWire)
```

Comment:
This is my concern.  If `'PRINTF_COND_` expanded into "x + y", then resulting expression is quite different from appearances.  I think this should be `if (('PRINTF_COND_) & ....`

Reply:
Fixed.